packages/jest-worker/src/workers/__tests__/processChild.test.ts

```diff
@@ -0,0 +1,340 @@
+/**
```

Comment:
would be nice to split up the change enough so git doesn't think this file is entirely new 😀